### PR TITLE
Badge design

### PR DESCRIPTION
**Pull request by felixfbecker**
Updates the badge design as in Figma and discussed with @AlicjaSuska over Zoom and extends the Storybook for them.
Also updates our `info` color to a lighter cyan (as in Figma).
See diff in Chromatic.

### CODE REVIEW

File: web/src/global-styles/global-styles.story.tsx [hunk 1]
```diff
@@ -159,52 +159,61 @@ add('Code', () => (
...
-                <div className="m-2 text-center" key={semantic}>
-                    <div className={`bg-${semantic} rounded`} style={{ width: '5rem', height: '5rem' }} />
-                    {semantic}
-                </div>
-            ))}
-        </div>
+            <h2>Semantic colors</h2>
+            <p>
+                These can be used to give semantic clues and always work both in light and dark theme. They are
+                availeble on most CSS components and the <code>border-</code> and <code>bg-</code> utility classes.
```
Comment (lguychard) [inline web/src/global-styles/global-styles.story.tsx:171]:
```suggestion
                available on most CSS components and the <code>border-</code> and <code>bg-</code> utility classes.
```

**Comment by felixfbecker** [comment]
I added a fallback for browsers that don't support `max()` yet, PTAL again

**Comment by twop** [comment]
Is there an easy way to see a link for Chromatic preview? Maybe there is a bot that can comment with a link?

**Comment by felixfbecker** [comment]
Yeah, clicking "Details" on this status check takes you straight to the Storybook:

image

"UI Tests" to the diff.

**Pull request merged**